README.md

```diff
@@ -122,26 +131,31 @@ You may now view pods in your cluster using `k9s -A` as below:
 
 ```
 $ solo init -t v0.42.5 -i node0,node1,node2 -n "${SOLO_NAMESPACE}" -s "${SOLO_CLUSTER_SETUP_NAMESPACE}" --key-format pfx 
+```
+Output
+```
 
 ******************************* Solo *********************************************
-Version                 : 0.22.1
-Kubernetes Context      : kind-solo
-Kubernetes Cluster      : kind-solo
-Kubernetes Namespace    : solo
+Version			: 0.22.1
+Kubernetes Context	: kind-solo
+Kubernetes Cluster	: kind-solo
+Kubernetes Namespace	: solo
 **********************************************************************************
-✔ Setup home directory and cache
-✔ Check dependencies [17s]
-  ✔ Check dependency: helm [OS: darwin, Release: 22.6.0, Arch: arm64] [7s]
-  ✔ Check dependency: keytool [OS: darwin, Release: 22.6.0, Arch: arm64] [17s]
-✔ Setup chart manager [1s]
-✔ Copy configuration file templates
-
+[33m❯[39m Setup home directory and cache
```

Comment:
these `[33m` and `[39m` look terrible on the readme previewer.  Have you looked into ways of removing this?

Reply:
I think the listr2 library has functionality for generating output for logs and systems that don't have interactive user prompts, this might be a solution.

Reply:
Unless there is a way to take the final snapshot of command output, like using a a terminal 
otherwise the output has all the log lines showing the progress ...

Reply:
I was thinking about adding an option for Solo to output in another format like the SimpleRenderer: https://listr2.kilic.dev/renderer/simple.html.  But, I think it might already be doing that.  

Another option is to use something like sed, awk, or grep to filter them out.  However, I tried and wasn't able to do it very quickly.  Perhaps, just log an issue with low priority and we can look at it later.

Reply:
I was able to remove lines contain these unwanted special character as well as escape color code,
but still the output no longer have indent to showing subtask status.

Before
```
✔ Setup home directory and cache
✔ Check dependencies
  ✔ Check dependency: helm [OS: darwin, Release: 23.3.0, Arch: arm64]
✔ Setup chart manager [0.9s]
✔ Copy configuration file templates
```

After
```
❯ Setup home directory and cache
✔ Setup home directory and cache
❯ Check dependencies
❯ Check dependency: helm [OS: darwin, Release: 23.3.0, Arch: arm64]
✔ Check dependency: helm [OS: darwin, Release: 23.3.0, Arch: arm64]
✔ Check dependencies
❯ Setup chart manager
✔ Setup chart manager
❯ Copy configuration file templates

```

Reply:
your change looks great, thanks!

Reply:
I think this is because listr2 is detecting that it is running inside a non user terminal and defaulting to the SimpleRenderer.  If you look at pipeline runs, it matches the 'After' format you listed above.  

example: https://github.com/hashgraph/solo/actions/runs/8559173056/job/23455469518?pr=199
```
✔ Copy configuration file templates
❯ Initialize
✔ Initialize
❯ Install chart 'fullstack-deployment'
✔ Install chart 'fullstack-deployment'
❯ Waiting for network pods to be ready
❯ Node: node0 (Pod: network-node0-0)
✔ Node: node0 (Pod: network-node0-0)
❯ Node: node1 (Pod: network-node1-0)
✔ Node: node1 (Pod: network-node1-0)
❯ Node: node2 (Pod: network-node2-0)
✔ Node: node2 (Pod: network-node2-0)
✔ Waiting for network pods to be ready
```